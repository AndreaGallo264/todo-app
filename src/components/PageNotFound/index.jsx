import React from "react";
import PageNotFoundImg from "../../assets/not-found.jpg";

const PageNotFound = () => {
  return (
    <img
      src={PageNotFoundImg}
      alt="Sorry we could not find the page you were looking for"
      style={{ width: "100%" }}
    />
  );
};

export default PageNotFound;
