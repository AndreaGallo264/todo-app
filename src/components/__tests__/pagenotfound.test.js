import React from "react";
import renderer from "react-test-renderer";
import PageNotFound from "../PageNotFound";

describe("Page not found component", () => {
  test("Matches snapshot", () => {
    const tree = renderer.create(<PageNotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
