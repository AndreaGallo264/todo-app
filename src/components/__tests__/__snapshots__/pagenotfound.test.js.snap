// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page not found component Matches snapshot 1`] = `
<img
  alt="Sorry we could not find the page you were looking for"
  src="not-found.jpg"
  style={
    Object {
      "width": "100%",
    }
  }
/>
`;
